_translator-files/po/de/Installation-for-Macintosh.po

```diff
@@ -65,7 +65,7 @@ msgstr "Updatest du? Es macht Sinn die [Konfiguration](Software-Manual#sicherung
 #, fuzzy
 #| msgid "[Download Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
 msgid "[Download Jamulus (Universal build)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}\\\\"
-msgstr "[Download Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
+msgstr "[Download Jamulus (Universal Build)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
```

Comment:
@pljones the fully German translation would be "Universeller Build" or "Universelle Version". @reinhardwh what would you prefer?